### ISSUE TITLE
Set values of non-existing data properties

### ISSUE THREAD

**Issue by akhramov** [epiloque/element-dataset#7]
Hi! Thank you for the polyfill.

As far as I understand, current implementation doesn't allow to set values of non-existing data properties. 

Consider the following HTML
```HTML
<h1 id="element-dataset-test">element-dataset</h1>
</div>
```

And JS:
```js
const testElement = document.getElementById('element-dataset-test');
testElement.dataset.userName = 'Jade Doe';
testElement.dataset.userName; // undefined using the polyfill, Jade Doe according to the specs:
 http://w3c.github.io/html/dom.html#domstringmap-__setter__
```

I support for this planned?

Thanks!

**Comment by epiloque** [comment]
I think this [should be working](https://github.com/epiloque/element-dataset/blob/master/test/element-dataset.spec.js#L21-L34), can you describe the environment?

**Comment by akhramov** [comment]
Yes, I saw the test suit. In fact, [this test](https://github.com/epiloque/element-dataset/blob/master/test/element-dataset.spec.js#L21-L27) would fail if `element` had no `data-date-of-birth` attribute declared in the markup.

I could check this locally, but I haven't figured out how to launch tests. Is there any guide or something?
